src/oidcCallback.jsx

```diff
@@ -12,7 +14,31 @@ export class oidcCallback extends React.Component {
   }
 
   errorCallback(e) {
-    browserHistory.replace('/')
+    this.props.dispatch(receiveError(e))
```

Comment:
Actually, in a connected component, if `mapDispatchToProps` isn't provided, it will similar pass through the dispatch function as a prop. I'd been meaning to clean up some places where we define and pass the function, but it only returns `dispatch`, so I tacked that on to this PR :)